src/FunctionApps/DSTP/src/test/kotlin/gov/cdc/prime/phdi/Utilities/HelperFunctionTest.kt

```diff
@@ -0,0 +1,108 @@
+package gov.cdc.prime.phdi.utilities
+
+import kotlin.test.assertEquals
+import kotlin.test.assertSame
+import kotlin.io.path.Path
+import org.junit.jupiter.api.Test
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.junit.MockitoJUnitRunner
+
+import java.net.URI
+import java.net.http.HttpClient
+import java.net.http.HttpRequest
+import java.net.http.HttpResponse
+
+class HelperFunctionsTest {
+    val singleMessage: String = javaClass.getResource("/singleMessage.hl7").readText()
+    val multipleMessages: String = javaClass.getResource("/multipleMessages.hl7").readText()
+    val mixedMessages: String = javaClass.getResource("/mixedMessages.hl7").readText()
+    val batchFileSingleMessage: String = javaClass.getResource("/batchFileSingleMessage.hl7").readText()
+    val batchFileMultipleMessages: String = javaClass.getResource("/batchFileMultipleMessages.hl7").readText()
+    val batchFileNoMessage: String = javaClass.getResource("/batchFileNoMessage.hl7").readText()
+    val singleInvalidMessage: String = javaClass.getResource("/singleInvalidMessage.hl7").readText()
+    val elrToFhirMessage: String = javaClass.getResource("/known_outputs/singleELRToFHIRMessage.txt").readText()
+    val ecrToFhirMessage: String = javaClass.getResource("/known_outputs/singleECRToFHIRMessage.txt").readText()
+    val vxuToFhirMessage: String = javaClass.getResource("/known_outputs/singleVXUToFHIRMessage.txt").readText()
+    val invalidFhirMessage: String = javaClass.getResource("/known_outputs/invalidFHIRMessage.txt").readText()
+    val loremIpsum: String = javaClass.getResource("/loremIpsum.txt").readText()
+
+    // TEST READING FUNCTIONALITY
+    @Test
+    fun testReadHL7MessagesFromByteArray() {
+        val multipleMessages: ByteArray = 
+            javaClass.getResource("/multipleMessages.hl7").readBytes()
```

Comment:
probably should've been a little more clear - it looks like this is duplicating the assignment on line 18

Reply:
Ah, I see. Sorry, i misinterpreting the code suggestions. The `multipleMessages` variable is recreated on line 30 in order to read it in as bytes instead of a string. I'd typically just remove the `.readText()` call on line 18 and call the appropriate method where needed, but we only use the `ByteArray` version once, so it seemed cleaner to duplicate the read. Happy to change if that's not true though.